target/linux/ipq807x/image/generic.mk

```diff
@@ -23,6 +23,16 @@ define Device/UbiFit
 	IMAGE/nand-sysupgrade.bin := sysupgrade-tar | append-metadata
 endef
 
+define Device/redmi_ax6
+	$(call Device/FitImage)
+	$(call Device/UbiFit)
```

Comment:
You don't need these two as they are inherited from ax3600

Reply:
Should be fixed now.